convene-web/app/views/features/_locked_rooms.md

```diff
@@ -0,0 +1,23 @@
+Sometimes, people in a Room want a bit of privacy. Rooms can be _Locked_ by any
+Member of the Space. When locked, Rooms require a _passcode_ to enter. A Room's
+lock can be added, changed, or removed on the Room's configuration screen.
+
+## How To Lock a Room
+
+1. Open the Room's configuration.
+2. Check the "Lock this Room" checkbox.
+3. Enter a passcode.
+4. Save your changes.
+
+Locked Rooms show up with a "Lock" icon on the Space main page. People
+attempting to enter a Locked Room will be prompted to enter the Room's passcode.
+
+People who entered the Locked Room before may enter automatically, so long as
+the passcode has not changed since they last entered the room.
+
+## Some Caveats
+
+1. Locking a Room does not remove any Occupants already in the Room, but it will
+   require them to enter the passcode when they attempt to return to the Room.
+2. Unlocking a Room also requires the passcode. If you forget the password,
+   reach out to our support team and we'll unlock it for you.
```

Comment:
@colombene - How does this read from a technical-writing "how to use a feature" perspective?